### PR TITLE
Fix polyfill code

### PR DESCRIPTION
**Pull request by Offirmo**
obviously an error

### CODE REVIEW

**Comment by okuryu** [comment]
What error are you seeing? `require("intl")` provides the `IntlPolyfill` object into the global.
```
> require("intl")
{ __localeSensitiveProtos:
   { Number: { toLocaleString: [Function] },
     Date:
      { toLocaleString: [Function],
        toLocaleDateString: [Function],
        toLocaleTimeString: [Function] } },
  default: [Circular] }
> IntlPolyfill
{ __localeSensitiveProtos:
   { Number: { toLocaleString: [Function] },
     Date:
      { toLocaleString: [Function],
        toLocaleDateString: [Function],
        toLocaleTimeString: [Function] } },
  default: [Circular] }
```

File: views/pages/guides/runtime-environments.hbs [hunk 1]
```diff
@@ -99,7 +99,7 @@ if (global.Intl) {
     if (!areIntlLocalesSupported(localesMyAppSupports)) {
         // `Intl` exists, but it doesn't have the data we need, so load the
         // polyfill and replace the constructors with need with the polyfill's.
-        require('intl');
+        var IntlPolyfill = require('intl');
```
Comment (caridy) [inline views/pages/guides/runtime-environments.hbs]: this is not an error, this is just to guarantee that `intl` module executes before the rest of this code.

Comment (Offirmo) [inline views/pages/guides/runtime-environments.hbs]: I didn't see any error. Just reading the code, it was unclear as where the `IntlPolyfill` was coming from.

What is the use of a global variable here ? Global vars are considered... (you know what I mean)

Comment (caridy) [inline views/pages/guides/runtime-environments.hbs]: polyfills will just patch the runtime, they are not libraries.

Comment (ericf) [inline views/pages/guides/runtime-environments.hbs]: This polyfill happens to create the `IntlPolyfill` global variable when it's loaded.

**Comment by Offirmo** [comment]
I opened another PR in due place https://github.com/andyearnshaw/Intl.js/pull/135

**Comment by Offirmo** [comment]
confirmed by Intl developper https://github.com/andyearnshaw/Intl.js/pull/135#discussion_r39454067

**Pull request merged**